app/lib/service/download_counts/sync_download_counts.dart

```diff
@@ -180,10 +180,17 @@ Future<bool> processDownloadCounts(
   }
 }
 
-const numberOfSyncDays = 5;
+const defaultNumberOfSyncDays = 5;
```

Comment:
nit: `@visibleForTesting`

Reply:
I would like to use this value from an admin action, so I will keep it public.